ui/js/content.js

```diff
@@ -290,20 +290,26 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
      * @see manageaccess#initManageAccess
      */
     var getManageAccessData = function() {
-        return {
+        var widgetData = {
             'contextProfile': contentProfile,
             'messages': getManageAccessMessages(),
-            'defaultRole': contentProfile.resourceSubType === 'collabdoc' ? 'manager' : 'viewer',
-            'roles': {
-                'viewer': oae.api.i18n.translate('__MSG__CAN_VIEW__'),
-                'manager': oae.api.i18n.translate('__MSG__CAN_MANAGE__')
-            },
+            'defaultRole': 'viewer',
+            'roles': [
```

Comment:
This should have been done for groups, etc. as well. I'll see if I can push a fix in time for the bash